src/Playwright.TestingHarnessTest/tests/mstest.spec.ts

```diff
@@ -0,0 +1,214 @@
+import { test, expect } from './baseTest';
+
+test('should be able to forward DEBUG=pw:api env var', async ({ runTest }) => {
+  const result = await runTest({
+    'ExampleTests.cs': `
+      using System;
+      using System.Threading.Tasks;
+      using Microsoft.Playwright.MSTest;
+      using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+      namespace Playwright.TestingHarnessTest.MSTest;
+
+      [TestClass]  
+      public class <class-name> : PageTest
+      {
+          [TestMethod]
+          public async Task Test()
+          {
+              await Page.GotoAsync("about:blank");
+              await Page.SetContentAsync("<button disabled>Click me</button>");
+              try
+              {
+                  await Page.Locator("button").ClickAsync(new() { Timeout = 1_000 });
+              }
+              catch
+              {
+              }
+          }
+      }`,
+    '.runsettings': `
+      <?xml version="1.0" encoding="utf-8"?>
+      <RunSettings>
+        <RunConfiguration>
+          <EnvironmentVariables>
+            <DEBUG>pw:api</DEBUG>
+          </EnvironmentVariables>
+        </RunConfiguration>
+      </RunSettings>`,
+  }, 'dotnet test --settings=.runsettings');
```

Comment:
This seems pretty painful to author, edit, and debug. Instead of putting string inline, consider changing `runTest` to just take a path to a directly with all these files. That way, your editor can help with the verbose XML and C#.

Our JS test harness suffers the same issue, but for some reason I don't mind it in JS, but I do mind it here. Perhaps it's because there's not so much extra syntax.